src/components/Homepage.js

```diff
@@ -0,0 +1,14 @@
+import { NavLink } from "react-router-dom";
+
+function Homepage() {
+  return (
+    <>
+    <h1>Welcome</h1>
+      <NavLink to="/beers">All Beers</NavLink>
+      <NavLink to="/random-beer">Random Beer</NavLink>
+      <NavLink to="/new-beer">New Beer</NavLink>
+    </>
```

Comment:
The Route path MUST match menu links.

- In app.js you say that in this route  ` /random ` `<Route path='/random' element={ <RandomBeer />} />` , you will render according component.
- HOwever your menu link takes you to `/random-beer `--> which has not content inside


```
     <Route path='/Beers' element={ <Beers />}></Route>
     <Route path='/:id' element={ <BeerDetails />}></Route>
     <Route path='/random' element={ <RandomBeer />} />
     <Route path='/new' element={ <NewBeer />}></Route>
```